extra/3-magic-8-ball.js

```diff
@@ -45,10 +45,39 @@
 
 // This should log "The ball has shaken!"
 // and return the answer.
+const answers = [
```

Comment:
Try to break this array into four array and then put them into one. e.g 
veryPositive = [];
positive = [];
You can use spread operator
answers[...verypositive, ...positive]